test/consistency.jl

```diff
@@ -381,6 +381,65 @@ function consistent_functions(nlps; rtol=1.0e-8, exclude=[])
         end
       end
     end
+    
+    if !(jth_hess_coord in exclude)
+      k = m
+      Ls = Vector{Any}(undef, N)
+      for i = 1:N
+        V = jth_hess_coord(nlps[i], x, y, k)
+        I, J = hess_structure(nlps[i])
```

Comment:
It seems we assume all Hessians have the same structure? That's a constraint that should be well documented.

Reply:
We assume that the structure is the same than the Hessian Lagrangian, and so in coo formal the vectors have length `nnzh`.
This is now document in the functions `jth_hess_coord!` and `jth_hess`.
In the actual implementation of the function for ADNLPModel and SlackModel, we assert it with `@lencheck nlp.meta.nnzh vals`